src/index.js

```diff
@@ -1,11 +1,58 @@
-import _ from "lodash";
+import './style.css';
 
-function component () {
-  const element = document.createElement('div');
+const element = document.querySelector('#todolist');
+const toDoList = [
+  {
+    description: 'wash the dishes',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'complete To Do list project',
+    completed: false,
+    index: 1,
+  },
+];
 
-  element.innerHTML = _.join(['hello', 'webpack'], ' ');
+const getToDoList = () => {
+  if (toDoList === null) {
+    return [];
+  }
+  return toDoList;
+};
 
-  return element;
-}
+// eslint-disable-next-line consistent-return
+const printList = () => {
+  const list = toDoList;
+  let tasks = '';
+  for (let i = 0; i < list.length; i += 1) {
+    // eslint-disable-next-line no-loop-func
+    list.forEach((task) => {
+      if (task.index === i) {
+        tasks += `
+        <li id = "${task.index}" class="item">
+          <label>
+            <input class="checked" type="checkbox" name="chk${task.index}" id="chk${task.index}">
+            <input class="edit borderless fit hidden" type="text" name="edit${task.index}" id="edit${task.index}" placeholder="Edit task...">
+            <p id="task${task.index}">${task.description}</p>
+          </label>
+          <div class="edit-manager flex hidden">
+            <button class="edit-confirm">Confirm</button>
+            <button class="edit-cancel">Cancel</button>
+          </div>
+          <div class="list-editor flex">
+            <button class="edit-task">edit</button>
+            <button class="delete-task">Del</button>
+          </div>
+        </li>

```

Comment:
- [ ] @Swarzstein great work on your project however the design doesn't match the recommended one check the comparison below:

Design Given          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/206860490-69f5542c-44c3-4bdd-95c7-2ff91040eab2.png) |  ![image](https://user-images.githubusercontent.com/61408860/206860525-61b10066-930d-4b5d-b7c4-81ba3b28d0d2.png)

Reply:
- [ ] @Swarzstein as previously requested kindly ensure the design matches the design given.

Reply:
Literally I have seen other projects and non of them have been rejected because of that.